### DISCUSSION TITLE
关于UDP映射传输空闲后自动关闭问题

### DISCUSSION THREAD

**Discussion by Zightch** [natfrp/wiki#268] [使用问题] [answered]
我有一个自己写做的多人联机游戏，使用UDP协议  
我在服务端里设置了1分钟的心跳包  
在心跳包发送之前的通讯一切正常  
中间大概有40~50秒的空闲时间没有通讯  
服务端发心跳包后  
客户端好像并没有收到也没有回复服务端  
是映射的UDP空闲之后自动关掉了吗  
如果是这样，那空闲多长时间关闭呢  
如果我可以设置我的心跳包时间更短  
请给和好的建议

**Comment by fakeboboliu** [comment, marked as answer]
您好，

目前超时时间是 30秒，
考虑到您给出的场景，我们将在下个版本中调整到 2分钟。

感谢您的反馈。

**Reply by Zightch** [reply]
好的  
谢谢你的回答
